packages/devops/configs/vm-app/.bashrc

```diff
@@ -0,0 +1,126 @@
+# ~/.bashrc: executed by bash(1) for non-login shells.
```

Comment:
Flagging that I haven't reviewed this as it looks template?

Reply:
I believe so yeah, not sure where Igor got it.